apple/internal/rule_factory.bzl

```diff
@@ -637,6 +632,11 @@ fashion, such as a Cocoapod.
                 doc = """
 A list of iOS app clips to include in the final application bundle.
 """,
+            ),
+            "codesignopts": attr.string_list(
```

Comment:
I moved this in this RFC to restrict this attr to only ios_application. Not saying that's good but otherwise it would require much more threading

Reply:
is this technically a breaking change?

Reply:
def. not particularly suggesting we do this, but when I made it I was thinking about the balance of realistic use cases vs maintainability

Reply:
If something is needed by ios_application, it minimally is needed by ios_app_clip as well, and most likely the app extensions too. I think we need to keep it where it was 😕.